src/System.ComponentModel.TypeConverter/src/System/ComponentModel/TypeDescriptor.cs

```diff
@@ -2665,7 +2662,8 @@ public static void SortDescriptorArray(IList infos)
         [Conditional("DEBUG")]
         internal static void Trace(string message, params object[] args)
         {
-            Debug.WriteLineIf(s_traceDescriptor.Enabled, string.Format(CultureInfo.InvariantCulture, "TypeDescriptor : {0}", string.Format(CultureInfo.InvariantCulture, message, args)));
+            //TODO leaving this in for convenience; remove before release?
```

Comment:
Can you rephrase this to "// TODO: Determine the value this tracing adds" or something more as to what the TODO is for?